### PR TITLE
Remove the "not-enabled" flag from AMM topics and update Known Amendments page

### PR DESCRIPTION
**Pull request by amarantha-k**
When merged, this PR closes issue #2486

### CODE REVIEW

**Review by oeggert — APPROVED** [review]
Approving, but there are a couple not-enabled tags still in the `ledger_data` and `ledger_entry` ledger methods pages.

File: resources/known-amendments.md [hunk 1]
```diff
@@ -210,8 +210,8 @@ Adds several new transaction result codes.
 | Amendment    | AMM |
 |:-------------|:----|
 | Amendment ID | 8CC0774A3BF66D1D22E76BBDA8E8A232E6B6313834301B3B23E8601196AE6455 |
-| Status       | Open for Voting |
-| Default Vote (Latest stable release) | No |
+| Status       | Enabled |
+| Default Vote (Latest stable release) | Yes |
```
Comment (tequdev) [inline resources/known-amendments.md]: Doesn't the Default Vote match rippled’s Feature.cpp?
スクリーンショット 2024-03-23 9 57 21

**Review by mDuo13 — CHANGES_REQUESTED** [review]
As tequ noted, the default vote hasn't changed.

File: resources/known-amendments.md (hunk 1 above)
Comment (amarantha-k) [reply resources/known-amendments.md]: Thanks for catching this. Fixed.

**Review by tequdev — APPROVED** [review]
👍

**Pull request merged**